test/utils/Defaults.sol

```diff
@@ -454,6 +455,17 @@ contract Defaults is Constants, Merkle {
         return baseParams(users.admin, asset, EXPIRATION, MERKLE_ROOT);
     }
 
+    function baseParams(string memory name, IERC20 asset_) public view returns (MerkleBase.ConstructorParams memory) {
```

Comment:
we dont' need this function

Reply:
I introduced this to avoid collisions in create2.